cmd/utils/flags.go

```diff
@@ -45,6 +45,9 @@ const (
 	c_RPCFlagPrefix     = "rpc."
 	c_PeersFlagPrefix   = "peers."
 	c_MetricsFlagPrefix = "metrics."
+
+	c_regionPortOffset = 1
+	c_zonePortOffset   = 199
```

Comment:
So, for a start port of 1000, I would get:
```
prime: 1000
regions: 1001-1129
zones: 1199-1228
```

I'd rather they were contiguous, just to minimize the port range we occupy in someone's NAT/firewall/proxy

Reply:
This PR doesn't aim to change the actual port definitions themselves, just the way they're calculated

Reply:
But if we want to do this in the future feel free to make an issue